Pod/Classes/SEGMixpanelIntegration.m

```diff
@@ -23,6 +23,18 @@ - (instancetype)initWithSettings:(NSDictionary *)settings andMixpanel:(Mixpanel
     return self;
 }
 
+-(instancetype)initWithSettings:(NSDictionary *)settings launchOptions:(NSDictionary *)launchOptions
```

Comment:
You're not calling this method anywhere. The factory has to supply the launch options to this method for this to work.